Rubberduck.VBEEditor/Events/VBENativeServices.cs

```diff
@@ -268,7 +214,7 @@ private static ICodePane GetCodePaneFromHwnd(IntPtr hwnd)
             }
         }
 
-        private static IWindow GetWindowFromHwnd(IntPtr hwnd)
+        public static IWindow GetWindowFromHwnd(IntPtr hwnd)
```

Comment:
Ditto for the `IWindow`. Maybe even consider make `ICodePane` and `IWindow` implement a `ISubclassable` interface?

Reply:
Ditto above. `DesignerWindowSubclass` is eventually going to need an `IWindow` member. `ISubclassable` might not be a bad idea, but that implementation feeds into the fragmentation of the code due to having both a VBA and VB6 assembly mirroring one another. Also, in that the message pumps are different, `ISubclassable` really isn't an accurate abstraction - `IWindow` should only be subclassed by a `DesignerWindowSubclass` and `ICodePane` should only be subclassed by a `CodePaneSubclass`.